setup.py

```diff
@@ -32,7 +32,13 @@ def read(fname):
 assert __version__ is not None
 
 REQUIREMENTS_DOCS = ["sphinx~=3.0", "alabaster~=0.7"]
-TESTS_BASIC = ["pytest>=5.0", "pytest-sugar", "coverage"]
+TESTS_BASIC = [
+    "pytest>=5.0",
+    "pytest-sugar",
+    "coverage",
+    "pytest-mypy-plugins; python_version<'3.10'",
```

Comment:
I don't completely grok the <3.10 rule here... why?

Reply:
Ah, good spot. It wasn't working on earlier 3.10 versions, but it seems OK now.